### ISSUE TITLE
Truecaller Permissions are granted which were set to IGNORE in AppOps

### ISSUE THREAD

**Issue by userdksingh** [RikkaApps/App-Ops-issue-tracker#116]
### What did you do? / 你做了什么？

Truecaller permissions are allowed after restart of device or after turning off developer mode. It automatically grants all permission in miui 12.5

### What did you expect to see? / 你期望看到什么？
I wanted to remain all the permissions which were set in AppOps app even after reboot.

**Comment by bgman111111** [comment]
Correct. Time I try it and it didn't work. In this time I didn't have GitHub account. And now I have GitHub account. This program is LIE. you put deny or ignore and it still send your data to ilegal spy. People's need to start protect this people's against government who don't know how to developer programa s
